mort_cal.py

```diff
@@ -0,0 +1,38 @@
+#fixed variables
+## Fixed variables are mostly unliked, as we don't have good reasons typically to make these
+## fixed vars. Usually, you want this passed in by a user or you want the application to read
+## the variables from a config file, where they can be modified instead
+interest_rate = 3.75
+property_tax = 0.0167
+numberPaymentsPerYear = 12
+term = 30
+totalMonths = numberPaymentsPerYear * term
+escrow = int(1046.88-257.36)
+principalPercent = .2
+affordableMortgage = 1500
+inputMessage = "Input New Home Price"
+successMessage = "Yay! You can afford this house!"
+failureMessage = "Bummer. Try again."
```

Comment:
A good portion of these values we should be passing into the program from the user. Interest rates, property taxes, number of payments per year, term, total months, escrow, principalPercent, affordableMortgage are all examples of a varying value. If you _HAVE_ to not bring these in from the user input, you should put them in a config file.